jobs/e2e_node/benchmark-config.yaml

```diff
@@ -26,24 +26,28 @@ images:
     image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
+    metadata: "user-data<test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
     tests:
       - 'create 35 pods with 0s? interval \[Benchmark\]'
   cosstable2-density2:
     image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
+    metadata: "user-data<test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
```

Comment:
I am wondering whether we should set this as default, and allow config file to overwrite.
Maybe we can file one PR to this. lt